examples/simple_examples/async_wss_web_server.py

```diff
@@ -0,0 +1,59 @@
+import os
+import uuid
+import random
+import json
+from aiohttp import web
+from panini import app as panini_app
+from examples.simple_examples._wss_manager import WSSManager, html
+
+
+app = panini_app.App(
+    service_name='async_NATS_WSS_bridge',
+    # host='nats-server' if 'HOSTNAME' in os.environ else '127.0.0.1',
+    host='54.36.108.188',
```

Comment:
Should we leave such host, instead of simple 127.0.0.1 in examples?

Reply:
no, need to fix it